Information about Dedicated Generation 3 (#1535)
platformsh/platformsh-docs: docs/src/configuration/services/postgresql.md

See the [Upgrading to PostgreSQL 12 with `postgis`](#upgrading-to-postgresql-12-with-the-postgis-extension) section below for more details.
{{< /note >}}

{{< note >}}

> PostgreSQL is not available in the dedication plan.

Reviewer: ```suggestion
PostgreSQL is not available on Dedicated Production Clusters.
```

> {{< /note >}}

Reviewer: ```suggestion
{{< /note >}}

```
